### PR TITLE
dev-lang/mujs: bump to version 1.0.5

### PR DESCRIPTION
**Pull request by ConiKost**
Closes: https://bugs.gentoo.org/show_bug.cgi?id=668492
Closes: https://bugs.gentoo.org/show_bug.cgi?id=669854
Closes: https://bugs.gentoo.org/show_bug.cgi?id=669884
Signed-off-by: Conrad Kostecki <conrad@kostecki.com>
Package-Manager: Portage-2.3.51, Repoman-2.3.11

### CODE REVIEW

File: dev-lang/mujs/mujs-1.0.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
```
Comment (hsoft) [inline dev-lang/mujs/mujs-1.0.5.ebuild]: IIRC, idiomatic EAPI declaration is unquoted.

**Comment by hsoft** [comment]
Looks good. will fully review soon.

File: dev-lang/mujs/mujs-1.0.5.ebuild (hunk 1 above)
Comment (ConiKost) [reply dev-lang/mujs/mujs-1.0.5.ebuild]: Ok. I've seen both ways.

**Comment by mmokrejs** [comment]
This https://github.com/gentoo/gentoo/pull/10283 has discarded tghe changes from https://github.com/gentoo/gentoo/pull/4229 . So again I get on Gentoo::Prefix installations:

```
>>> Install mujs-1.0.5 into /apps/gentoo/var/tmp/portage/dev-lang/mujs-1.0.5/image category dev-lang
make -j4 DESTDIR=/apps/gentoo/var/tmp/portage/dev-lang/mujs-1.0.5/image/apps/gentoo install-shared libdir=/usr/lib64 prefix=/usr VERSION=1.0.5 install-static 
make build=release shared
make[1]: Entering directory '/apps/gentoo/var/tmp/portage/dev-lang/mujs-1.0.5/work/mujs-1.0.5'
make[1]: Nothing to be done for 'shared'.
make[1]: Leaving directory '/apps/gentoo/var/tmp/portage/dev-lang/mujs-1.0.5/work/mujs-1.0.5'
install -d /apps/gentoo/var/tmp/portage/dev-lang/mujs-1.0.5/image/apps/gentoo/usr/include
install -d /apps/gentoo/var/tmp/portage/dev-lang/mujs-1.0.5/image/apps/gentoo/usr/lib64
install -d /apps/gentoo/var/tmp/portage/dev-lang/mujs-1.0.5/image/apps/gentoo/usr/lib64/pkgconfig
install -d /apps/gentoo/var/tmp/portage/dev-lang/mujs-1.0.5/image/apps/gentoo/usr/bin
install -m 644 mujs.h /apps/gentoo/var/tmp/portage/dev-lang/mujs-1.0.5/image/apps/gentoo/usr/include
install -m 644 build/release/mujs.pc /apps/gentoo/var/tmp/portage/dev-lang/mujs-1.0.5/image/apps/gentoo/usr/lib64/pkgconfig
install -m 755 build/release/mujs /apps/gentoo/var/tmp/portage/dev-lang/mujs-1.0.5/image/apps/gentoo/usr/bin
install -m 755 build/release/libmujs.so /apps/gentoo/var/tmp/portage/dev-lang/mujs-1.0.5/image/apps/gentoo/usr/lib64
install -m 644 build/release/libmujs.a /apps/gentoo/var/tmp/portage/dev-lang/mujs-1.0.5/image/apps/gentoo/usr/lib64
mv: cannot stat '/apps/gentoo/var/tmp/portage/dev-lang/mujs-1.0.5/image/usr/lib64/libmujs.so': No such file or directory
 * ERROR: dev-lang/mujs-1.0.5::gentoo failed (install phase):
 *   (no error message)
 * 
 * Call stack:
 *     ebuild.sh, line 124:  Called src_install
 *   environment, line 1560:  Called die
 * The specific snippet of code:
 *       mv -v "${D}"/usr/$(get_libdir)/lib${PN}.so{,.${PV}} || die;
```

**Comment by hsoft** [comment]
@mmokrejs It wasn't discarded. The `DESTDIR=${ED}` is still there. It looks like the added `mv` command is the culprit. I don't have a prefix setup to test with. Looks like it's time to set me up a prefix VM...

**Comment by mmokrejs** [comment]
Right, the 1.0.5 ebuild has been fixed in the tree (tested as of now) but 1.0.4 is still broken. Needs the same change to `mv -v "${ED}"...`. Thank you.

**Pull request closed without merging**